ajax/libs/vega-embed/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "vega-embed",
+  "filename": "vega-embed.min.js",
+  "version": "2.2.0",
+  "description": "Publish Vega visualizations as embedded web components with interactive parameters.",
+  "homepage": "https://github.com/vega/vega-embed",
+  "keywords": [
+    "vega",
+    "data",
+    "visualization",
+    "component",
+    "embed"
+  ],
+  "npmName": "vega-embed",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "vega-embed.js",
+        "vega-embed.min.js",
+        "vega-embed-schema.json"
```

Comment:
Because we don't need json file, we can use `vega-embed.*` instead of these three rules.